wordcab_transcribe/utils.py

```diff
@@ -60,7 +61,66 @@
     "₿",
 ]
 
+# run API for youtube video endpoint
+def run_api_youtube(url : str, sourceLang = "en", timestamps = "s", wordTimestamps = False):
```

Comment:
Two essential things for the functions:
* Never use CamelCase in Python. Could you go with the snake case for function parameters? Only a class has uppercase letters, e.g. `ASRService`.
* The functions are missing some parameters like `alignment`, `diarization`, or `dual_channel` for the audio file endpoint.